projects/feature-activation/0004-phased-testing.md

```diff
@@ -0,0 +1,206 @@
+- Feature Name: Feature Activation Phased Testing
+- Start Date: 2023-08-01
+- Initial Document: [Feature Activation](https://docs.google.com/document/d/1IiFTVW1wH6ztSP_MnObYIucinOYd-MsJThprmsCIdDE/edit)
+- Author: Gabriel Levcovitz <<gabriel@hathor.network>>
+
+# Summary
+[summary]: #summary
+
+This RFC describes a phased procedure for testing the Feature Activation project.
+
+# Motivation
+[motivation]: #motivation
+
+After all changes introduced by the previous Feature Activation RFCs are implemented, the mechanism should be tested with some "fake" features in both the testnet and mainnet environments, and only then we'll be able to use it with confidence in the mainnet to change real features like updating the maximum merkle path length, for example.
```

Comment:
I'd say "dull fatures" instead of "fake features", and maybe explain that dull features do not change any behavior and will be used exclusively to test the activation mechanism.

Reply:
Done in https://github.com/HathorNetwork/rfcs/pull/62/commits/2992fd5d8903b25c3a2605803d87b24e14d77603

Reply:
Thanks!